src/components/vm/overview/watchdog.jsx

```diff
@@ -68,7 +68,7 @@ const WatchdogModalAlert = ({ dialogError }) => {
     return <ModalError dialogError={dialogError.text} dialogErrorDetail={dialogError.detail} />;
 };
 
-export const WatchdogModal = ({ vm, isWatchdogAttached, idPrefix }) => {
+export const WatchdogModal = ({ vm, isWatchdogAttached, isPermanent, idPrefix }) => {
```

Comment:
Is "permanent" some libvirt lingo? Or could this be called `isRemovable`?

Reply:
No, I can change that to isRemovabel.